frontend/packages/shared/src/types/Commit.ts

```diff
@@ -0,0 +1,14 @@
+export interface Commit {
+  message: string;
+  author: CommitAuthor;
+  comitter: CommitAuthor;
+  sha: string;
+  messageShort: string;
+  encoding: string;
+}
+
+export interface CommitAuthor {
+  email: string;
+  name: string;
+  when: any;
```

Comment:
Kan denne være en string? 🤔 Eller kan den være mye forskjellig? 
```suggestion
  when: string;
```

Reply:
I backend er den en ```DateTimeOffset```

Reply:
Mistenker at vi får det som typen string, når vi ser på JSON strukturen. Vi burde kanskje mappe den fra string => Date? 🤔 Hva tenker du? 😄

Reply:
Jeg kan jo evt. bare kalle den ```formatDateToDateAndTimeString``` funksjonen rett inn i query funksjonen ```useRepoInitialCommitQuery```? Da slipper utviklere tenkte på å mappen den om der den brukes.

Reply:
Dersom vi **ALLTID** skal vise datoen i formatet til `formatDateToDateAndTimeString`, så er jeg enig i det. 😊 Hvis man ønsker at selve spørringen til backend skal være mer uavhengig av visnings-logikk, så burde den mappes om til en Date. `when: new Date(data.comitter.when)`. Så må UI/render funksjonen selv holde styr på hvordan datoen skal vises. Jeg skal ikke komme med noen sterke føringer her - tenker du kan bestemme det. 😊

Reply:
Kanskje bedre å ha den som Date, også la den som brukeren den sette formatet. Gjør det sånn for nå 😄